<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Match_Dialog</class>
 <widget class="QDialog" name="Match_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>989</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>90</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="home_team">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>231</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="visiting_team">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="home_score">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="visiting_score">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>400</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="home_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="visiting_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>591</width>
     <height>361</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Team</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Goals</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Got</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Diff</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Punkte</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="scorer">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>530</y>
     <width>451</width>
     <height>361</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nummer</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Team</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Goals</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Match_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Match_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
